var/spack/repos/builtin/packages/py-gimmik/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class PyGimmik(PythonPackage):
+    """Generator of Matrix Multiplication Kernels - GiMMiK - is a tool for generation of
+high performance matrix multiplication kernel code for various accelerator platforms."""
```

Comment:
```suggestion
    high performance matrix multiplication kernel code for various accelerator platforms."""
```
You can wrap this onto another line if it's too long